config/variables/CanadaPubSecALZ-main.yml

```diff
@@ -0,0 +1,78 @@
+variables:
+  var-hubnetwork-region: canadacentral
+  var-hubnetwork-managementGroupId: Connectivity
+  var-hubnetwork-azfw-configurationFileName: hub-azfw/hub-network.parameters.json
+  var-hubnetwork-nva-configurationFileName: hub-nva/hub-network.parameters.json
+  var-hubnetwork-subscriptionId: 4fd845de-f6c8-4e6d-9a87-c21c4ebf7edd
```

Comment:
Are those filled with our subscription id values and other information or is this the default?

Reply:
no this is the default , not our subscription id